app/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <attr format="reference" name="runtimeTheme" />
```

Comment:
I wanted to but there is none in the app :(. Regarding the size of the project, I'm not sure we'll get conflicts here :)